tests/rptest/services/redpanda.py

```diff
@@ -1317,6 +1321,15 @@ def raise_on_bad_logs(self, allow_list=None):
                             allowed = True
                             break
 
+                if "oversized allocation" in line:
+                    m = re.search("oversized allocation: (\d+) byte", line)
```

Comment:
Just checking because I always struggle with regexes. It not saying "bytes" at the end there is fine? This is a simple match?

Reply:
`search()` is the one that allows a match anywhere in the line, i.e., with arbitrary leading and trailing unmatched characters so this would match `... bytes`.